core/src/main/java/com/sequenceiq/cloudbreak/init/blueprint/BlueprintEntities.java

```diff
@@ -1,18 +1,10 @@
 package com.sequenceiq.cloudbreak.init.blueprint;
 
-import java.util.HashMap;
-import java.util.Map;
-
 import org.springframework.boot.context.properties.ConfigurationProperties;
 import org.springframework.context.annotation.Configuration;
 
 @Configuration
 @ConfigurationProperties(prefix = "cb.blueprint.cm")
-public class BlueprintEntities {
-
-    private Map<String, String> defaults = new HashMap<>();
+public class BlueprintEntities extends AbstractBlueprintEntities {
```

Comment:
just a hint, maybe better to rename to CM or ClusterBlueprintEntites, related to the config name like the dev version

Reply:
I do not want to rename this existing class

Reply:
all template is related to cm, anyway i do not want to touch existing logic, naming, this can be done in another task if needed